eventdiscovery-sdk/src/main/res/values-v21/theme_light.xml

```diff
@@ -1,8 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
 <resources>
-    <style name="SchedJoules_AppTheme.NoActionBar">
+
+    <style name="SchedJoules_BaseTheme.Light.NoActionBar">
```

Comment:
Nevermind this, I missed to see that there is `SchedJoules_Theme.Light` which is the parent of the `NoActionBar` version, so there is a short name for the top theme.